app/views/article-amp.html

```diff
@@ -447,6 +447,8 @@ <h1 class="related">Related articles</h1>
         <a subscriptions-action="subscribe" subscriptions-display="TRUE">Subscribe</a>
         or
         <a subscriptions-action="login" subscriptions-display="TRUE">Login</a>
+        <div subscriptions-action="subscribe" subscriptions-display="TRUE"
```

Comment:
Why do we need `subscriptions-display` at all in this case?

Reply:
because otherwise we wont run eval without `subscriptions-display` and so it'll be invisible.

Reply:
Yeah. Right. Would be good to find some other ideas around that in the future.